docs/standard/commandline/get-started-tutorial.md

```diff
@@ -0,0 +1,434 @@
+   * Adds the `read` command to the `quotes` command instead of to the root command.
+   * Creates `add` and `delete` commands and adds them to the `quotes` command.
+
+   The result is the following command hierarchy:
+
+   * Root command
+     * `quotes`
+       * `read`
+       * `add`
+       * `delete`
+
+   The app now implements the recommended pattern where the parent command (`quotes`)specifies an area or group, and its children commands (`read`, `add`, `delete`) are actions.
+
+   Global options are applied to the command and recursively to subcommands. Since `--file` is on the root command, it will be available automatically in all subcommands of the app.
+
+1. After the `SetHandler` code, add new `SetHandler` code for the new subcommands:
+
+   :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage3/Program.cs" id="sethandlers" :::
+
+   Subcommand `quotes` doesn't have a handler because it isn't a leaf command. Subcommands `read`, `add`, and `delete` are leaf commands under `quotes`, and `SetHandler` is called for each of them.
+
+1. Add the handlers for `add` and `delete`.
+
+   :::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage3/Program.cs" id="handlers" :::
+
+The finished app looks like this:
+
+:::code language="csharp" source="snippets/get-started-tutorial/csharp/Stage3/Program.cs" id="all" :::
+
+Build the project, and then try the following commands.
+
+Submit a nonexistent file to `--file` with the `read` command, and you get an error message instead of an exception and stack trace:
+
+```console
+scl read --file nofile
+```
+
+```output
+File does not exist
+```
+
+Try to run subcommand `quotes` and you get a message directing you to use `read`, `add`, or `delete`:
+
+```console
+scl quotes
+```
+
+```output
+Required command was not provided.
+
+Description:
+  Work with a file that contains quotes.
+
+Usage:
+  scl quotes [command] [options]
+
+Options:
+  --file <file>   An option whose argument is parsed as a FileInfo [default: sampleQuotes.txt]
+  -?, -h, --help  Show help and usage information
+
+Commands:
+  read                          Read and display the file.
+  delete                        Delete lines from the file.
+  add, insert <quote> <byline>  Add an entry to the file.
+```
+
+Run subcommand `add`, and then look at the end of the text file to see the added text:
+
+```console
+scl quotes add "Hello world!" "Nancy Davolio"
+```
+
+Run subcommand `delete` with search strings from the beginning of the file, and then look at the beginning of the text file to see where text was removed:
+
+```console
+scl quotes delete --search-terms David "You can do" Antoine "Perfection is achieved"
+```
+
+> [!NOTE]
+> If you're running in the *bin/debug/net6.0* folder, that folder is where you'll find the file with changes from the `add` and `delete` commands. The copy of the file in the project folder remains unchanged.
```

Comment:
do you want to add that this is because of the setting in the .csproj that says to copy it.